basics/uri.adoc

```diff
@@ -330,99 +565,424 @@ Remote UUris mean that the uThing is running on a different device and we need t
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 |        UENTITY_ID             |  UE_VERSION   |   UNUSED      |
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-|    ID_LEN     |        UAUTHORITY_ID (1=256 bytes)  ...       |
+|   NAME_LEN    |        UAUTHORITY_NAME  ...                   |
 |                                                               |
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 ----
 
-== URI Examples
+[%autowidth]
+|===
+|Field |Description
+
+|TYPE
+a| The type of authority contained in this micro URI. Occupies bits 8 through 15 (octet 1). +
+*MUST* be set to `0x03` for a URI with an authority.
 
-The following examples are based off the USubscription notification topic (see usubscription.proto).
+|NAME_LEN
+a|The length of the `UAUTHORITY_NAME`. Occupies bits 64 through 71 (octet 24). +
+*MUST* contain the length of the `UAUTHORITY_NAME` field in bytes.
 
+|UAUTHORITY_NAME
+a|The URI's authority. Occupies bits 72 through ( 72 + 8*NAME_LEN - 1 ). +
+*MUST* contain the UTF8 encoding of the UUri's `authority.name` property.
 
-.Local Uri
-[width="80%",cols="30%,70%",options="header"]
 |===
-|Format | Value
 
-|Long
-|`/core.usubscription/3/SubscriptionChange#Update`
+NOTE: The authority is not padded to the next 32 bit word.
+--
 
-|Short
-| `/0/3/32768`
+=== Examples
 
-|Micro
-|`[1, 0, -128, 0, 0, 0, 3, 0]`
+The examples below can be used as test vectors for implementing client libraries.
 
+[%autowidth]
 |===
+|authority |entity |version |resource |Short form URI
+
+| none
+| `0x0000`
+| `0x03`
+| `0x8000`
+l|[
+  0x01, // version
+  0x00, // type
+  0x80, 0x00, // UResource ID
+  0x00, 0x00, // UEntity ID
+  0x03, // UEntity major version
+  0x00, // unused
+]
+
+l|`1G1YZ23J9P5800001`
```

Comment:
```suggestion
l|1G1YZ23J9P5800001
```

Reply:
hmm, the VIN also is a literal value. We format the hex values accordingly so, for consistency, I'd rather keep this as is. Alternatively we can remove the formatting for all values ...